.travis.yml

```diff
@@ -1,40 +1,33 @@
 sudo: required
 language: go
 go:
-    - "1.12"
-
+- '1.12'
 services:
-  - docker
-
+- docker
 install: true
-
-script: 
-  - VERBOSE=true make test int 
-  - ./travis-ci.sh 
-
+script:
+- export BRANCH=$(if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then echo $TRAVIS_BRANCH; else echo $TRAVIS_PULL_REQUEST_BRANCH; fi)
+- echo "TRAVIS_BRANCH=$TRAVIS_BRANCH, PR=$TRAVIS_PULL_REQUEST, BRANCH=$BRANCH"
```

Comment:
Added these 2 for clarity; very unintuitive how `$TRAVIS_BRANCH` won't be the PRs branch. See the issues/article I linked to elsewhere on this PR.